storage/innobase/include/ut0ut.h

```diff
@@ -305,6 +305,33 @@ operator<<(
 	return(lhs);
 }
 
+/** This is a wrapper class, used to print any number in IEC style */
+struct bytes_iec {
+ explicit bytes_iec(unsigned long long t): m_val(t) {}
+ const unsigned long long m_val;
+};
+
+/** Like hex operator above, except for bytes_iec */
+inline
+std::ostream&
+operator<<(
+	std::ostream&	lhs,
+	const bytes_iec&	rhs)
```

Comment:
Please avoid excessive line breaks, for `grep` friendliness:
```c++
inline std::ostream &operator<<(std::ostream &lhs, const bytes_iec &rhs)
```
The only constraint is the maximum line length of 79 characters, and it is not being exceeded here.

Reply:
Is an `inline` implementation really a good idea? This would seem to slow down the compilation and bloat the executable size. The code is not time-critical (the messages are output very rarely), so the function should better be defined in a `.cc` file.